src/Orchard/Orchard.Framework.csproj

```diff
@@ -419,7 +427,9 @@
     <Compile Include="Services\ClientHostAddressAccessor.cs" />
     <Compile Include="Services\DefaultJsonConverter.cs" />
     <Compile Include="Services\IClientHostAddressAccessor.cs" />
+    <Compile Include="Services\IYamlParser.cs" />
```

Comment:
Oh! Where is it? The only "YAML parser" I could find was the one used by the ExtensionHarvester, but it's not general-purpose. Perhaps you are referring to the one used in O2? If so, should we use that one instead?